src/caf/distribute/gravity_model/multi_area.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 """Implementation of a self-calibrating single area gravity model."""
 # Built-Ins
+from __future__ import annotations
+
 import functools
 import logging
 import os
 from dataclasses import dataclass
 from pathlib import Path
-from typing import Any, Optional
+from typing import Any, Optional, Iterator
```

Comment:
For future reference its better to import modules directly rather than specific functions / classes as it keeps the namespace cleaner and also avoids issues in sphinx docs e.g. `import pathlib` instead of `from pathlib import Path`

Typing is the one exception as the convention for that module is to import the classes directly, to avoid sphinx docs picking these up they can be imported with private aliases e.g. `from typing import Any as _Any`